client/src/main.rs

```diff
@@ -179,19 +179,9 @@ fn main() {
                     let mut nonce = _api.get_nonce().unwrap();
                     for account in accounts.into_iter() {
                         let to = get_accountid_from_str(account);
-                        #[allow(clippy::redundant_clone)]
-                        let xt: UncheckedExtrinsicV4<_> = compose_extrinsic_offline!(
-                            _api.clone().signer.unwrap(),
-                            Call::Balances(BalancesCall::transfer(
-                                GenericAddress::Id(to.clone()),
-                                PREFUNDING_AMOUNT
-                            )),
-                            nonce,
-                            Era::Immortal,
-                            _api.genesis_hash,
-                            _api.genesis_hash,
-                            _api.runtime_version.spec_version,
-                            _api.runtime_version.transaction_version
+                        let xt = _api.balance_transfer(
```

Comment:
nono, this won't work!
The faucet should be able to batch fund multiple accounts without always awaiting the next block. So we need to increase the nonce manually, as the api-client can't know the next nonce

Reply:
Ah yes, that totally makes sense. It didn't work, as the `Call::Balances` enum is wrong for the parachain case, which why I made the change in the first place, but I did not realize why it was implemented like that.

In that case, I will change it back and use the `compose_call!` macro instead.

Reply:
will this be fixed?
```                                                                                                                             
Faucet drips to 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y (Alice's nonce=5)                                                                                                  
thread 'client' panicked at 'extrinsic error code 1010: Invalid Transaction: Transaction has a bad signature', /home/abrenzikofer/.cargo/git/checkouts/substrate-api-client-9327f54164a9792a/a6e1616/src/rpc/client.rs:173:33
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', /home/abrenzikofer/.cargo/git/checkouts/substrate-api-client-9327f54164a9792a/a6e1616/src/lib.rs:454:45
```